app/models/participant_declaration.rb

```diff
@@ -64,6 +64,8 @@ class ParticipantDeclaration < ApplicationRecord
 
   scope :ect, -> { where(participant_profile_id: ParticipantProfile::ECT.select(:id)) }
   scope :mentor, -> { where(participant_profile_id: ParticipantProfile::Mentor.select(:id)) }
+  scope :ects, -> { where(type: "ParticipantDeclaration::ECT") }
+  scope :mentors, -> { where(type: "ParticipantDeclaration::Mentor") }
```

Comment:
I think we can replace the first two now that we have type column with ECT/Mentor